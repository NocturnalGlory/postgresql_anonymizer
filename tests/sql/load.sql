CREATE EXTENSION IF NOT EXISTS anon CASCADE;

SELECT anon.isloaded() IS FALSE; 

SELECT anon.load();

SELECT anon.isloaded() IS TRUE;

SELECT anon.unload();

SELECT anon.isloaded() IS FALSE;

DROP EXTENSION anon CASCADE;
