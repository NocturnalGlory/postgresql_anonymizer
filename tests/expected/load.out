CREATE EXTENSION IF NOT EXISTS anon CASCADE;
NOTICE:  installing required extension "tsm_system_rows"
SELECT anon.isloaded() IS FALSE; 
 ?column? 
----------
 t
(1 row)

SELECT anon.load();
 load 
------
 t
(1 row)

SELECT anon.isloaded() IS TRUE;
 ?column? 
----------
 t
(1 row)

SELECT anon.unload();
 unload 
--------
 t
(1 row)

SELECT anon.isloaded() IS FALSE;
 ?column? 
----------
 t
(1 row)

DROP EXTENSION anon CASCADE;
